project/Dependencies.scala

```diff
@@ -66,11 +68,11 @@ object Dependencies {
   }
 
   object Pureconfig {
-    private val version = "0.17.8"
-    val Pureconfig = "com.github.pureconfig" %% "pureconfig" % "0.17.8"
+    private val version = "0.17.9"
```

Comment:
tidied up versions here - they seem to be released together so all module should share the version